core/common/src/main/java/alluxio/security/authentication/DefaultAuthenticationServer.java

```diff
@@ -122,8 +123,15 @@ public void unregisterChannel(UUID channelId) {
       try {
         channelInfo.getSaslServer().dispose();
       } catch (SaslException e) {
-        LOG.warn("Failed to dispose sasl client for channel-Id: {}. Error: {}", channelId,
-            e.getMessage());
+        LogUtils.warnWithException(LOG,
+            String.format("Failed to dispose sasl client for channel-Id: {}", channelId), e);
+      }
+
+      try {
+        channelInfo.getSaslServerDriver().onCompleted();
+      } catch (Exception e) {
+        LogUtils.warnWithException(LOG, String.format(
```

Comment:
Remove `String.format`: `LogUtils.warnWithException(LOG, "Failed to complete the authentication session for channel-Id: {}", channelId, e);`